packages/server/src/app/challenge/validators/challenge.validator.js

```diff
@@ -0,0 +1,36 @@
+export const newChallengeSchema = {
+  name: {
+    notEmpty: { bail: true, errorMessage: 'Een naam is verplicht.' },
+    isLength: {
+      options: { min: 3, max: 255 },
+      errorMessage: 'Een naam moet tussen de 3 en 80 tekens bevatten.'
+    },
+    escape: true
+  },
+  start_date: {
+    notEmpty: { bail: true, errorMessage: 'Een startdatum is verplicht.' },
+    isDate: { bail: true, errorMessage: 'Een startdatum moet een geldige datum zijn' },
+    escape: true
+  },
+  end_date: {
+    notEmpty: { bail: true, errorMessage: 'Een einddatum is verplicht.' },
+    isDate: { bail: true, errorMessage: 'Een einddatum moet een geldige datum zijn' },
+    escape: true
+  },
+  groups: {
```

Comment:
A group should be nullable?

Reply:
Yes, because you can also have a challenge with yourself